clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -1018,6 +1035,34 @@ Value LowerFunction::rewriteCallOp(const LowerFunctionInfo &CallInfo,
 
       break;
     }
+    case ABIArgInfo::Indirect:
+    case ABIArgInfo::IndirectAliased: {
+      assert(NumIRArgs == 1);
+      // TODO(cir): For aggregate types
```

Comment:
For all these things we need to add new MissingFeatures and assert on them (just like you did for existing ones a bit below)

Reply:
done!